test/algorithms/set_operations/intersection/test_intersection.hpp

```diff
@@ -179,13 +196,12 @@ typename bg::default_area_result<G1>::type test_intersection(std::string const&
 #if ! defined(BOOST_GEOMETRY_TEST_ONLY_ONE_TYPE)
     if (! settings.debug)
     {
-        // Check _inserter behaviour with stratey
-        typedef typename bg::strategy::intersection::services::default_strategy
-            <
-                typename bg::cs_tag<point_type>::type
-            >::type strategy_type;
+        // Check inserter behaviour with stratey
+        using strategy_type
+            = typename bg::strategies::relate::services::default_strategy<G1, G2>::type;
```

Comment:
This was a left-over related to the change to umbrella strategies.
Because the define `BOOST_GEOMETRY_TEST_ONLY_ONE_TYPE`, this code was effectively not tested for some years... but the new unit test does not use that define.